src/object-properties.js

```diff
@@ -132,7 +132,8 @@ const properties = {
  * @typedef {object} NavigationAction
  * @property {'nextSheet'|'prevSheet'|'firstSheet'|'lastSheet'|'goToSheet'|'goToSheetById'|'goToSheetById'|'goToStory'|'openWebsite'|'odagLink'|'openChainedApp'} action - Name of navigation action
  * @property {string=} appId - app ID. Required for 'openChainedApp'
- * @property {string=} sheet - sheet ID. Required for 'goToSheet' and 'goToSheetById'
+ * @property {string=} sheet - sheet - sheet ID. Required for 'goToSheet' and 'goToSheetById'
```

Comment:
Here it says that there is a goToSheetById as well, we are not adding the go to chart option there?